api-tests/src/steps/cri-steps.ts

```diff
@@ -336,3 +418,38 @@ When(
     );
   },
 );
+
+function assertNoUnexpectedJarProperties(
```

Comment:
We've now got a mixture of `function` and arrow function definitions - this is definitely a matter of taste, but it would be nice to be consistent. At the moment pretty much everything else is arrow functions...